test/journeys/specs/tap/widget-space/oneOnOne.js

```diff
@@ -52,48 +53,15 @@ describe(`Widget Space: One on One: TAP`, () => {
   before(`pause to let test users establish`, () => browser.pause(5000));
 
   before(`inject token for spock`, () => {
-    local.browser.execute((localAccessToken, localToUserEmail) => {
-      const options = {
-        accessToken: localAccessToken,
-        onEvent: (eventName, detail) => {
-          // eslint-disable-next-line object-shorthand
-          window.ciscoSparkEvents.push({eventName: eventName, detail: detail});
-        },
-        toPersonEmail: localToUserEmail,
-        initialActivity: `message`
-      };
-      window.openSpaceWidget(options);
-    }, spock.token.access_token, mccoy.email);
+    loginAndOpenWidget(local.browser, spock.token.access_token, true, mccoy.email);
     local.browser.waitForExist(`[placeholder="Send a message to ${remote.displayName}"]`, 30000);
   });
 
   before(`open remote widget for mccoy`, () => {
-    remote.browser.execute((localAccessToken, localToUserEmail) => {
-      const options = {
-        accessToken: localAccessToken,
-        onEvent: (eventName, detail) => {
-          // eslint-disable-next-line object-shorthand
-          window.ciscoSparkEvents.push({eventName: eventName, detail: detail});
-        },
-        toPersonEmail: localToUserEmail,
-        initialActivity: `message`
-      };
-      window.openSpaceWidget(options);
-    }, mccoy.token.access_token, spock.email);
+    loginAndOpenWidget(remote.browser, mccoy.token.access_token, true, spock.email);
     remote.browser.waitForExist(`[placeholder="Send a message to ${local.displayName}"]`, 30000);
   });
 
-  it(`loads the test page`, () => {
-    const title = browserLocal.getTitle();
-    assert.equal(title, `Widget Space Production Test`);
-  });
-
-  it(`loads the user's name`, () => {
-    local.browser.waitForVisible(`h1.ciscospark-title`);
-    local.browser.waitUntil(() => local.browser.getText(`h1.ciscospark-title`) !== `Loading...`);
-    assert.equal(local.browser.getText(`h1.ciscospark-title`), remote.displayName);
-  });
-
```

Comment:
Is there a reason for deleting these two tests? I don't see a check for correct title/display name in the `loginAndOpenWidget` function

Reply:
I guess I could add them back, I didn't really see a good reason for keeping them. They were originally there for a confirmation that the browser was ready, but that is being handled in the login function now

Reply:
Ok, works for me. Don't imagine that there would be a bug where the display name is wrong that wouldn't be caught by the check against `Send message to ${remote.displayName}` anyways.